### PR TITLE
Match the search box .

### PR DESCRIPTION
**Pull request by 13nehu**
screenshot from 2017-04-17 18-53-10

screenshot from 2017-04-17 18-53-04
#142

### CODE REVIEW

File: src/app/results/results.component.css [hunk 1]
```diff
@@ -32,6 +32,20 @@
   color: #88aaaa;
   padding-left: 60px;
 }
+.container-fluid ul{
+  padding-top: 16px;
+}
+
+.container-fluid li{
+  height: 15px;
```
Comment (harshit98) [inline src/app/results/results.component.css]: Indent this line correctly.

**Comment by harshit98** [comment]
@13nehu Try to solve whole issue. I have gone through the issue, and some of it features has not been implemented : 
-  font and font size in search box.
-  sub menu
- adjust the position "Tools" to match the sample service

**Comment by mariobehling** [comment]
Please provide a link to a test page on Github pages as well. Thanks.

File: src/app/search-bar/search-bar.component.css [hunk 2]
```diff
@@ -62,4 +78,4 @@
   .align-search-btn{
     left: -5%;
   }
-}
+}
```
Comment (daminisatya) [inline src/app/search-bar/search-bar.component.css]: CRLF leave a line.

Comment (13nehu) [inline src/app/search-bar/search-bar.component.css]: @harshit98  i was not able to get any idea where sub menu is defined in the code . rest changes have been implemented .

**Comment by isuruAb** [comment]
`git config --global user.email "your-email@example.com"` try this with your git email address @13nehu   .

**Comment by mariobehling** [comment]
I am merging this as an exception, so we can progress.

**Pull request merged**